### PR TITLE
[ParticleMechanics] Correcting Errors in Nightly Build

### PR DESCRIPTION
**Pull request by bodhinandach**
The ParticleMechanicsApplication's nightly build fails. This corrects the error @loumalouomega @philbucher

### CODE REVIEW

File: applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def __init__(self, model, custom_settings):
         self.settings.ValidateAndAssignDefaults(default_settings)
 
         # Construct the linear solvers
-        import linear_solver_factory
+        from KratosMultiphysics import python_linear_solver_factory as linear_solver_factory
```
Comment (philbucher) [inline applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py]: this is already part of #3974 
I would prefer if you exclude it from this PR such that I don't have to solve conflicts many times.
Thx!

(and sorry for breaking your stuff)

**Review by loumalouomega — DISMISSED** [review]
Ok for me

File: applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py (hunk 1 above)
Comment (bodhinandach) [reply applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py]: Okay, I will take them out. Or can you merge first? I will resolve the conflict.

Comment (bodhinandach) [reply applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py]: I revert back

**Review by philbucher — APPROVED** [review]
:+1: 
Sorry for breaking

**Pull request merged**